### ISSUE TITLE
Detect unstable connections

### ISSUE THREAD

**Issue by bisand** [bisand/node-red-contrib-tibber-api#54]
Improvement suggested by @sungtroll in issue #50 

> If i can suggest improvement is would be add a special settings for users with unstable 4G internet to do someting like:
> if tibber-feed not receiving data, then check if local internet connection lost as first step.
> like ping 8.8.8.8 or someting else.
> Then if lost, wait for local internet connection is back before stopping and restart connection to tibber API
> if restored connection but not receiving data from tibber API, then disconnect and reconnect to tibber API
> 
> This will make it quicker to get the live feed after internet connection is restored
> 
> Maybe a test connection to tibber demo account is someting to consider to check if tibber API is working or not. It probably have no restrictions of number of concurrent connections.

**Comment by Sungtroll** [comment]
I want to mention sometimes but very rearly data is lost between AMS and tibber pulse.
More often data is lost between tibber pulse and tibber backend specially if it is sent by 4G router. Maybe tibber should keep sending timestamps with no data or "tibber not receiving data" to let tibber-feed know everytihing is working fine between tibber and tibberfeed so it dont reconnects because of missing data when the problem is at the other end. Dagfinn is the developer of pulse and he probably can improve the feed if it is neccessery. 

Some replys i got from dagfinn:

problem: tibber sending duplicate messages. (old reply but duplicates startet again 1. nov 2021, i sent new bug report to dagfinn yesterday)

Dagfinn says Aidon does not send timestamps, tibber must create their own timestams when receiving data from pulse.

original message:
Takk,
Det var ein bug i backend. Påverka Aidon målarar då dei ikkje har eigne timestamps.
Her er litt av problemet at Aidon ikkje har timestamps på dataen sin. Då må vi lage timestamps på pakkane sjølv når dei lande på serveren. 

problem lost packets:
Dagfinn, recommends using usb power for critical applications because low power from aidon can make pulse unsable

tibber pulse does not buffer data, when it is sent but not received at backend it is lost.

**Comment by Sungtroll** [comment]
For your information i send this old copy of Tibbers solution on to many concurrent sockets that never closes:
i will delete it later because it does not belong in this thread.

Tibber wrote:
Omstart av service, så fikset det problemet :) 

On Wed, Jan 13, 2021 at 01:56 PM,  I wrote: 
hva var feilen?
On Wed, Jan 13, 2021 at 01:51 PM, I wrote:
nå er det i orden
On Wed, Jan 13, 2021 at 01:44 PM, Tibber wrote:
Har feilsøkt litt på api tilkoblignen og ser at det er noko ustabilitet der no. Våre utviklere ser på dette no :) 
On Wed, Jan 13, 2021 at 01:33 PM, Tibber wrote: 
Det kan vere fleire grunner til at APIet er brukt opp. Kan ta fra minuter til timer før systemet ser at du har kun 1 session igjen og dermed åpner opp kontakten igjen. 
On Wed, Jan 13, 2021 at 01:24 PM, Tibber wrote: 
øke antall sockets er noko vi må vurdere i framtiden, men pr dags dato er det 2 session pr kunde som er limiten. 
Kjører du kun dette eller har du fleire oppkoblinger? 
Too many concurrent socket tyder på at du har fleire en 2 aktive tilkoblinger. da må du lukke tilkoblingene og vente ca 1 time. 
On Wed, Jan 13, 2021 at 01:10 PM, Tibber wrote: 
max 2 session.
On Wed, Jan 13, 2021 at 01:09 PM, I wrote: 
hvor mange concurrent sockets kan man ha og når blir de frakoblet på serveren hos dere?
On Wed, Jan 13, 2021 at 12:56 PM, Tibber wrote: 
Heisann,
Da må du lukke alle åpene session og starte igjen. 
On Wed, Jan 13, 2021 at 12:48 PM, I wrote:
hei. jeg får beskjed om to many concurrent sockets for token
